### ISSUE TITLE
unable to get version 

### ISSUE THREAD

**Issue by richkifvel** [bridgecrewio/yor-action#241]
This action is failing because the version isn't being passed to the yor cmd:

/home/runner/work/_temp/24d83e68-7a04-40f8-aede-e0b4a4a7d8d8/yor -v
Incorrect Usage. flag provided but not defined: -v

```
NAME:
   yor - enrich IaC files with tags automatically
...

GLOBAL OPTIONS:
   --help, -h  show help (default: false)
Error: /10 13:39:06 [ERROR] flag provided but not defined: -v
```

Whereas last week we had no problems passing the version in:

```
/home/runner/work/_temp/bfbb8a10-6409-4937-b8c0-64e603643f32/yor -v
yor version 0.1.180

  __    __
  \ \  / /
   \ \/ /___  _  ____
    \  /  _ \| |/  __|
    | |  |_| |   /
    |_|\____/|__|v0.1.180
 Yor Findings Summary
 Scanned Resources:	  1[14]
 New Resources Traced: 	  0
 Updated Resources:	  0


/usr/bin/git status -s --untracked-files=no
Nothing has changed
```

I don't see any new action releases in this time and our workflow hasn't changed for a couple months. Do you know why this is happening?

Our workflow is very basic:
```
jobs:
  yor-job:
    runs-on: ubuntu-latest
    steps:
      - name: Checkout repo
        uses: actions/checkout@v3
        with:
          token: ${{ secrets.token }}
          fetch-depth: 0
      - name: Run yor action
        uses: bridgecrewio/yor-action@main
        with:
          tag_groups: git
          skip_tags: **redacted**
          commit_changes: false
```

**Comment by samgiles** [comment]
Workaround is to set the version to 0.1.180:

```
      - name: Run yor action
        uses: bridgecrewio/yor-action@main
        with:
          version: 0.1.180
```

**Comment by gruebel** [comment]
was fixed with `0.1.182`
